contextualized/functions.py

```diff
@@ -1,7 +1,9 @@
 import torch
 import torch.nn.functional as F
 
-
+zero_vector = lambda x, *args: torch.zeros((len(x), 1))
```

Comment:
Names should relate to regularization losses, maybe `no_reg`? 
Also compile link functions and regularization losses into lists `LINK_FUNCTIONS` AND `REGULARIZERS` for clarity

Reply:
I don't have strong feelings either way; putting the organization in `__init__.py` exports and highlights the available utilities.